kex-runner/src/main/kotlin/org/vorpal/research/kex/jacoco/minimization/GreedyTestReductionImpl.kt

```diff
@@ -56,4 +59,7 @@ class GreedyTestReductionImpl : TestSuiteMinimizer {
 
         return importantTests
     }
+   private fun String.getPath(): Path {
```

Comment:
I think now we can remove this function and an import of `com.jetbrains.rd.util.first` as they are not used anymore

Reply:
fixed that